crates/tree-sitter-owl-ms/queries/highlights.scm

```diff
@@ -8,74 +8,71 @@
 "," @punctuation.delimiter
 ; "o" @punctuation.delimiter
 
-"integer" @keyword
-"decimal" @keyword
-"float" @keyword
-"string" @keyword
-"Prefix:" @keyword
-"Ontology:" @keyword
-"Import:" @keyword
-"Annotations:" @keyword
-"Datatype:" @keyword
-"EquivalentTo:" @keyword
-"Class:" @keyword
-"SubClassOf:" @keyword
-"EquivalentTo:" @keyword
-"DisjointWith:" @keyword
-"DisjointUnionOf:" @keyword
-"HasKey:" @keyword
-"ObjectProperty:" @keyword
-"Domain:" @keyword
-"Range:" @keyword
-"SubPropertyOf:" @keyword
-"InverseOf:" @keyword
-"Characteristics:" @keyword
-"SubPropertyChain:" @keyword
-"DataProperty:" @keyword
-"AnnotationProperty:" @keyword
-"Individual:" @keyword
-"Types:" @keyword
-"Facts:" @keyword
-"SameAs:" @keyword
-"DifferentFrom:" @keyword
+[
+  (keyword_integer)
+  (keyword_decimal)
+  (keyword_float)
+  (keyword_string)
+  (keyword_import)
+  (keyword_annotations)
+  (keyword_datatype)
+  (keyword_equivalent_to)
+  (keyword_class)
+  (keyword_sub_class_of)
+  (keyword_equivalent_to)
+  (keyword_disjoint_with)
+  (keyword_disjoint_union_of)
+  (keyword_has_key)
+  (keyword_object_property)
+  (keyword_domain)
+  (keyword_range)
+  (keyword_sub_property_of)
+  (keyword_equivalent_to)
+  (keyword_disjoint_with)
+  (keyword_inverse_of)
+  (keyword_sub_property_chain)
+  (keyword_functional)
+  (keyword_inverse_functional)
+  (keyword_reflexive)
+  (keyword_irreflexive)
+  (keyword_symmetric)
+  (keyword_asymmetric)
+  (keyword_transitive)
+  (keyword_data_property)
+  (keyword_characteristics)
+  (keyword_annotation_property)
+  (keyword_individual)
+  (keyword_types)
+  (keyword_facts)
+  (keyword_same_as)
+] @keyword
```

Comment:
Some keywords seem to have been lost. E.g. `Ontology:`, `Prefix:`, `EquivalentClasses:`

Reply:
Added 8 more keywords.